### PR TITLE
sci-biology/aghermann: initial ebuild

### PR DESCRIPTION
**Pull request by hmmr**
Initial packaging for aghermann, a sleep-research experiment manager

Aghermann is a program designed around a common workflow in
sleep-research, complete with scoring facility; cairo subpixel
drawing on screen or to file; conventional PSD and EEG Micrcontinuity
profiles; Independent Component Analysis; artifact detection; and
Process S simulation following Achermann et al, 1993.

### CODE REVIEW

File: sci-biology/aghermann/aghermann-1.0.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+DEPEND="x11-libs/gtk+:3
+	dev-libs/libconfig
+	dev-libs/libunique:3
+	sci-libs/gsl
+	sci-libs/itpp
+	sci-libs/fftw
+	dev-libs/libconfig
+	dev-libs/libxml2
+	media-libs/samplerate
```
Comment (marbre) [inline sci-biology/aghermann/aghermann-1.0.3.ebuild]: media-libs/samplerate does not exist. Do you mean media-libs/libsamplerate?

File: sci-biology/aghermann/aghermann-1.0.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2013 Gentoo Foundation
```
Comment (marbre) [inline sci-biology/aghermann/aghermann-1.0.3.ebuild]: Please adjust the year.

File: sci-biology/aghermann/aghermann-1.0.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+src_configure() {
+        econf \
+                --bindir="${EPREFIX}"/bin
+}
+
+src_install() {
+	emake DESTDIR="${D}" install || die "make install failed"
+	dodoc AUTHORS ChangeLog* README || die "dodoc failed"
+}
```
Comment (marbre) [inline sci-biology/aghermann/aghermann-1.0.3.ebuild]: Looks like the `default`: https://devmanual.gentoo.org/ebuild-writing/functions/src_install/index.html

Further, no die is needed, as emake and dodoc die on their own.

File: sci-biology/aghermann/aghermann-1.0.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+	 dev-lang/lua"
+
+DEPEND="${RDEPEND}"
+
+src_configure() {
+        econf \
+                --bindir="${EPREFIX}"/bin
+}
+
+src_install() {
```
Comment (junghans) [inline sci-biology/aghermann/aghermann-1.0.3.ebuild]: Drop `src_install`, the next two lines are the default on `EAPI=5`.

File: sci-biology/aghermann/aghermann-1.0.3.ebuild [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+inherit eutils
+
+DESCRIPTION="A sleep-research experiment manager, EDF viewer & Achermann's Process S model runner"
+HOMEPAGE="http://johnhommer.com/academic/code/aghermann"
+SRC_URI="http://johnhommer.com/academic/code/aghermann/source/${PN}-${PV}.tar.xz"
```
Comment (junghans) [inline sci-biology/aghermann/aghermann-1.0.3.ebuild]: Use `${P}` instead of `${PN}-${PV}`.

File: sci-biology/aghermann/aghermann-1.0.3.ebuild [hunk 6]
```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+inherit eutils
```
Comment (junghans) [inline sci-biology/aghermann/aghermann-1.0.3.ebuild]: `eutils` isn't used!

**Comment by hmmr** [comment]
I admit I mostly borrowed the initial ebuild from the handbook and having seen that it worked, just went with that. Hopefully this time, it should be it.

File: sci-biology/aghermann/aghermann-1.0.3.ebuild [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+RDEPEND="x11-libs/gtk+:3
+	 dev-libs/libconfig
+	 dev-libs/libunique:3
+	 sci-libs/gsl
+	 sci-libs/itpp
+	 sci-libs/fftw
+	 dev-libs/libconfig
+	 media-libs/libsamplerate
+	 x11-libs/vte
+	 dev-lang/lua"
```
Comment (jlec) [inline sci-biology/aghermann/aghermann-1.0.3.ebuild]: Only use tab for indention. And when you are at it, please sort the deps alphabetically (that's not required, but I like it ;) )

**Comment by jlec** [comment]
@junghans Please check the travis ouput for information on that PR before merging

    virtual.suspect               1
       sci-biology/aghermann/aghermann-1.0.3.ebuild: DEPEND: consider using 'virtual/pkgconfig' instead of 'dev-util/pkgconfig'

Also inspect commit messages. The once created with repoman always carry the "portage ...." tag.

**Pull request merged**